x-pack/plugins/apm/scripts/bootstrap_apm/README.md

```diff
@@ -0,0 +1,56 @@
+# Getting started
+
+A script for creating an elastic stack deployment on a remote cluster using `ecctl` and generating APM data using [apm-synthtrace](https://github.com/elastic/kibana/tree/main/packages/elastic-apm-synthtrace) tool.
+
+## Prerequisites for running the script
+
+- [ecctl](https://www.elastic.co/downloads/ecctl) installed
+- API key
+  - You can generate a key on [staging](https://staging.found.no/deployment-features/keys)
+- Make sure you have installed Kibana dependencies
+
+## Configuration
+
+Update the following configuration with your own credentials and store it in your home directory `$HOME/.ecctl/config.json`
```

Comment:
It's probably best to have folks bootstrap the confguration through `ecctl init`
 https://www.elastic.co/guide/en/ecctl/current/ecctl-configuring.html